task/04-arrays-tasks.js

```diff
@@ -249,7 +252,8 @@ function getMovingSum(arr) {
  * [ "a" ] => []
  */
 function getSecondItems(arr) {
-   throw new Error('Not implemented');
+   let value = true;
+   return arr.filter(() => value = !value);
```

Comment:
see https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

filter also has an index argument, try to do it without `value` variable